### ISSUE TITLE
[med] JAWS+Chrome: Many Pages: The required fields are not marked as required

### ISSUE THREAD

**Issue by ogumen** [Expensify/App#5463]
## Action Performed:
1. Using JAWS+Chrome, open https://staging.new.expensify.com/
2. Press Tab key to focus the 'Enter your phone or email' field.
3. Note if the field is visually marked or announced as required.

## Expected Result:
The required fields should be visually marked as required with the asterisks next to the field labels and the legend explaining the meaning of the asterisk either at the top of the form or the word 'required' next to the visible field labels.
Also the required fields should be announced as required by the screen reader.

## Actual Result:
The required field is not visually marked as required, it is not announced as required either.

HTML:
`<div class="css-1dbjc4n" style="margin-top: 12px;"><div dir="auto" class="css-901oao" style="color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 13px; font-weight: 700; line-height: 18px; margin-bottom: 8px;">Enter your phone or email:</div><input autocapitalize="none" autocorrect="off" placeholder="Phone or Email" autocomplete="email" dir="auto" spellcheck="false" type="text" class="css-11aywtz r-tz77pi" value="" style="background-color: rgb(255, 255, 255); border-color: rgb(236, 236, 236); border-radius: 8px; border-width: 1px; color: rgb(11, 27, 52); font-family: GTAmericaExp-Regular; font-size: 15px; height: 42px; padding: 10px 12px; vertical-align: middle;"></div>`

Other occurrences:
1. Set password -> 'Enter a password', the field is not visually marked as required.
2. Payments -> 'Your Paypal username', the field is not visually marked as required.
3. Profile -> 'First name', and 'Last name' fields are not visually marked as required.

## Workaround:
No workaround

## Area issue was found in:
Landing page, Set password page, Add payment method flow, Profile overlay

## Failed WCAG checkpoints
3.3.2

## User impact:
The users are not aware of which fields are required.

## Suggested resolution:
Ensure that the required fields are visually marked as required either with an asterisk next to the field labels and the legend explaining the asterisk meaning at the top of the form, or by using the word 'required' next to the visible field labels. See: https://www.w3.org/WAI/WCAG21/Techniques/html/H90.html

Additionally, use the aria-required attribute to announce the field as required.
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-required_attribute

## Platform:
Where is this issue occurring?

- Web

**Version Number:**
**Reproducible in staging?:**Yes
**Reproducible in production?:**
**Notes/Photos/Videos:** Any additional supporting documentation
**Issue reported by:**

https://user-images.githubusercontent.com/88733897/134565873-66eacee3-1b7c-4e76-915f-c932ff01b5f2.mp4

![Bug5180580_Required_form_field](https://user-images.githubusercontent.com/88733897/134565888-ffa5aea3-0d58-44fb-8e84-7cbfa5e767f8.png)
![Bug5180580_required_-_set_password](https://user-images.githubusercontent.com/88733897/134565899-dcb1d587-50c2-4877-9e38-9f8a92f0868c.png)
![Bug5180580_Required_-_your_paypal_username_-_payments](https://user-images.githubusercontent.com/88733897/134565907-d4e55cf1-4140-4238-83a6-47c3aaae3766.png)
![Bug5180580_required_fields_-_profile](https://user-images.githubusercontent.com/88733897/134565920-d411a12c-8347-4662-aff4-7fd54087c60b.png)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)
